app/services/github_service.rb

```diff
@@ -1,13 +1,25 @@
 class GithubService
+
   def conn
     Faraday.new(url: 'https://api.github.com') do |faraday|
-      # faraday.headers['Authorization'] = ENV['GITHUB_API_KEY']
-      faraday.headers['Authorization'] = "token 484cabdee3b4e5de4226ed80470600b274e435d9"
+      faraday.headers['Authorization'] = ENV['GITHUB_API_KEY']
     end
   end
 
-  def repos_body
-    response = conn.get('/user/repos')
+  def get_url(url)
+    response = conn.get("/user/#{url}")
     JSON.parse(response.body, symbolize_names: true)
   end
+
+  def repos
+    get_url('repos')
```

Comment:
these methods are quite small and repetitive, but I think it will make more sense to call `.repos` instead of `.get_url("repos")`

Reply:
good point